RevenueCatUI/Templates/V2/Components/Tabs/TabsComponentView.swift

```diff
@@ -80,10 +81,11 @@ struct LoadedTabsComponentView: View {
     private var tabControlContext: TabControlContext
 
     @State
-    private var tierPackageContexts: [PackageContext]
+    private var tierPackageContexts: [String: PackageContext]
 
-    var activeTabViewModel: TabViewModel {
-        return self.viewModel.tabViewModels[self.tabControlContext.selectedIndex]
+    var activeTabViewModel: TabViewModel? {
+        return self.viewModel.tabViewModels[self.tabControlContext.selectedTabId] ??
+            self.viewModel.tabViewModels.values.first
```

Comment:
Make `activeTabViewModel` optional so we never actually crash, and instead we just don't render the `LoadedTabComponentView` below if the model cannot be found.